nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -227,7 +227,7 @@ done
 mkdir -p /lib
 ln -s @modulesClosure@/lib/modules /lib/modules
 ln -s @modulesClosure@/lib/firmware /lib/firmware
-echo @extraUtils@/bin/modprobe > /proc/sys/kernel/modprobe
+echo @extraUtils@/bin/modprobe-kernel > /proc/sys/kernel/modprobe
```

Comment:
```suggestion
# see comment in stage-1.nix for explanation 
echo @extraUtils@/bin/modprobe-kernel > /proc/sys/kernel/modprobe
```

Reply:
Done. Commit will be part of the nex tpush.